app/routers/shoplist_export.py

```
@@ -146,9 +146,11 @@ def _register_font_if_available() -> str:
             if FONT_NAME not in pdfmetrics.getRegisteredFontNames():
                 pdfmetrics.registerFont(TTFont(FONT_NAME, str(FONT_PATH)))
             return FONT_NAME
-        except Exception:
+        except Exception as exc:
```

Ruff (0.13.3)

Do not catch blind exception: `Exception`

(BLE001)